.reuse/dep5

```diff
@@ -106,6 +106,10 @@ Files: clients/github-graphql/src/main/assets/schema.graphql
 Copyright: 2017 Gregor Martynus
 License: MIT
 
+Files: evaluator/src/main/resources/osadl/osadl-matrix.csv
```

Comment:
I don't see this file in the change set anymore - it seems like it was replaced by osadl-matrix-with-explanations.json - so t guess this has to be updated here as well

Reply:
Thanks for pointing this out! Done.